src/Jenssegers/Mongodb/Relations/EmbeddedBy.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Jenssegers\Mongodb\Relations;
+
+use Illuminate\Database\Eloquent\Collection;
+use Jenssegers\Mongodb\Relations\BelongsTo;
+
+class EmbeddedBy extends BelongsTo
+{
+    /**
+     * Set the base constraints on the relation query.
+     *
+     * @return void
+     */
+    public function addConstraints()
+    {
+        if (static::$constraints) {
+            $this->query->where($this->getOwnerKey(), '=', $this->related->{$this->getOwnerKey()});
+        }
+    }
+
+    /**
+     * Set the constraints for an eager load of the relation.
+     *
+     * @param  array  $models
+     * @return void
+     */
+    public function addEagerConstraints(array $models)
+    {
+
```

Comment:
throw an exception here?

Reply:
Looks like Jensy leaves this method blank too so I think it's ok as is. I was about to add exceptions to methods that aren't implemented but realised I don't actually know what's already working (and required) as it is, so I think we might just have to be aware that there could be strange issues around sub documents if we try to do any more than we're currently using them for.

Reply:
Good thinking.